rio/subsystems/armsubsystem.py

```diff
@@ -102,7 +102,11 @@ def extension(self, speed):
         This is the spool motor, it controls
         how much rope we let out.
         """
-        self.elevatorMotor.set(speed)
+        if self.elevatorEncoder.getPosition() >= 100 and speed < 0.1:
```

Comment:
What is this evaluation for? This doesn't seem like a good idea to include here.

If you feel you need it, keep it in. Everything else I'm ok with.